.github/workflows/stale.yml

```diff
@@ -0,0 +1,28 @@
+name: Mark stale issues and pull requests
+
+on:
+  schedule:
+  - cron: "30 1 * * *"
```

Comment:
Should we add `- workflow_dispatch` for the possibility to manually trigger the execution ?

Reply:
@rerowep Why not. Can we have both? Through a cron *and* manually triggered?

Reply:
I've made the change, base on this example: https://github.com/actions/stale/blob/main/.github/workflows/stale.yml